libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -153,9 +157,29 @@ - (void)userAccountManager:(SFUserAccountManager *)userAccountManager
 }
 
 #pragma mark - Private methods
+- (SFUserAccount *) createInterimTestUser:(SFSDKTestCredentialsData *) credsData  {
+    SFUserAccount *newAcct = [[SFUserAccountManager sharedInstance]createUserAccount];
+    newAcct.accountIdentity.orgId = @"__TESTORGID__";
```

Comment:
This is an area where, if we staged the credentials first, then applied them to the creation of the user account—which is more like what we'd support at runtime anyway—we could do away with setting `accountIdentity` values explicitly.